solana/gatekeeper-lib/src/service/GatekeeperService.ts

```diff
@@ -88,21 +89,20 @@ export class GatekeeperService {
    * @returns {Promise<string>}
    */
   private async signAndSerializeBuiltTransaction(
-    transaction: Transaction,
-    blockhash?: string
+    transaction: Transaction
   ): Promise<string> {
     // recent blockhash and feepayer are required for signing and serialization
-    transaction.recentBlockhash = blockhash
-      ? blockhash
-      : (await this.connection.getRecentBlockhash()).blockhash;
+    transaction.recentBlockhash = (
+      await this.connection.getRecentBlockhash()
```

Comment:
If this RPC call fails or doesn't contain a `blockhash` property, the lambda fails?

Reply:
this library has no concept of lambda but yes, the function will throw an error